packages/ui/src/components/Transactions/Transaction.tsx

```diff
@@ -29,6 +31,13 @@ const Transaction = ({
   const isProxy = useMemo(() => isProxyCall(aggregatedData.name), [aggregatedData])
   // FIXME this is duplicated
   const appliedClass = useMemo(() => (isProxy ? 'blue' : 'red'), [isProxy])
+  const { selectedNetworkInfo } = useNetwork()
+
+  const onOpenLink = useCallback(() => {
+    const encodedRpc = encodeURIComponent(selectedNetworkInfo?.rpcUrl || '')
+    const link = `https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=${encodedRpc}#/extrinsics/decode/${aggregatedData.callData}`
+    window.open(link, '_blank')
+  }, [aggregatedData, selectedNetworkInfo])
```

Comment:
Why do we need a callback for that? And make a window open instead of using the regular link?
f.e:
```js
function getLink() {
const encodedRpc = encodeURIComponent(selectedNetworkInfo?.rpcUrl || '')
    return `https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=${encodedRpc}#/extrinsics/decode/${aggregatedData.callData}`
}
```

Reply:
No particular reason, just used to it. I've set the link in a variable instead